src/theia/sfm/reconstruction_builder.h

```diff
@@ -146,6 +146,9 @@ class ReconstructionBuilder {
                        const std::string& image2,
                        const ImagePairMatch& matches);
 
+  // Assign a mask for each image.
+  bool SetMasksForFeaturesExtraction(const std::vector<std::string>& mask_filepaths);
```

Comment:
You are right.
Moreover, in the .cpp, the following line is over 80 chars:

`return feature_extractor_and_matcher_->SetMasksForFeaturesExtraction(mask_filepaths);`

How could we cut it?